gaugan/models/blocks.py

```diff
@@ -1,80 +1,67 @@
 import tensorflow as tf
-import tensorflow_addons as tfa
+from tensorflow.keras import Sequential
 from tensorflow.keras import layers, initializers
+import tensorflow_addons as tfa
 
-from .spade import SpatialAdaptiveNormalization
+from .spade import SPADE
 
 
 class ResidualBlock(layers.Layer):
-    """Residual Block
-
-    Reference: https://arxiv.org/abs/1903.07291
-
-    Args:
-        n_filters (int): Number of filters in the conv layers
-    """
-
-    def __init__(self, n_filters: int, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.n_filters = n_filters
+    def __init__(self, filters, **kwargs):
+        super().__init__(**kwargs)
+        self.filters = filters
```

Comment:
We can also accept a `config` argument (of type `ml_collections.ConfigDict`) and access all the hyperparameters from there.